etc/bastion/osh-http-proxy.conf.dist

```diff
@@ -1,90 +1,79 @@
 ############################################################################################
-# Config for the HTTP Proxy of The Bastion.
-# This is a JSON file, its syntax must be valid at all times. To verify:
-# => grep -v ^# /etc/bastion/osh-http-proxy.conf|python -mjson.tool>/dev/null && echo OK
-#
-# If you're on a production bastion you can verify it can properly load its configuration:
-# => perl -I/opt/bastion/lib/perl -MOVH::Bastion -e 'die OVH::Bastion::load_configuration_file(file => "/etc/bastion/osh-http-proxy.conf")'
+## Config for the HTTP Proxy of The Bastion.
```

Comment:
double # here

Reply:
It's necessary for the script parsing it to generate the doc ;)